models/articles.models.js

```diff
@@ -48,3 +48,30 @@ exports.getCommentsWidArticleId = (articleId) => {
       return comments;
     });
 };
+
+exports.addComment = (articleID, newComment) => {
+  const { username, body } = newComment;
+  const commentProps = ["username", "body"];
+
+  const isProps = commentProps.reduce((username, body) => {
+    return username && body in newComment;
+  }, true);
+
+  if (!isProps || Object.keys(newComment).length > 2) {
+    return Promise.reject({
+      status: 400,
+      msg: "Please pass correct comment object!",
+    });
```

Comment:
this is unnecessary, an object that has all the correct fields should return a 201 and add object to db but ignore anything other than what is required.